lte/cloud/go/services/subscriberdb/storage/per_sub_digest.go

```diff
@@ -0,0 +1,118 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package storage
+
+import (
+	"magma/lte/cloud/go/lte"
+	lte_protos "magma/lte/cloud/go/protos"
+	"magma/lte/cloud/go/services/subscriberdb/protos"
+	"magma/orc8r/cloud/go/blobstore"
+	"magma/orc8r/cloud/go/mproto"
+	"magma/orc8r/cloud/go/storage"
+	merrors "magma/orc8r/lib/go/errors"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
+)
+
+const (
+	perSubDigestBlobstoreNetworkKey = "per_sub_digest_network"
```

Comment:
Add note that this is a placeholder network, since we're gonna use the network as the key

Also suffix with "_internal"